view/frontend/templates/map.phtml

```diff
@@ -1,10 +1,16 @@
-<div id="map"></div>
-<script type="text/javascript">// <![CDATA[
-    var apiKey = '<?php echo $this->getConfig('key'); ?>';
-    require([
-        'LCB_GoogleMap/js/map'
-    ], function (map) {
-        map.render(
+<?php if($this->isSnazzyMapsEnabled()){ ?>
+
+    <?php echo $this->getSnazzyMapIframe();?>
+
+<?php }else{ ?>
```

Comment:
@Jigsaw00 please remove unnecessary php tags from there and format the part }else

git commit --amend and then push force